spec/controllers/forms_controller_spec.rb

```diff
@@ -139,6 +143,17 @@
             submit
             expect(response).to redirect_to thank_you_form_url(@form)
           end
+          it 'updates the current_user object' do
+            [:email, :first_name, :last_name].each do |attribute|
+              expect(@current_user.send(attribute))
```

Comment:
It is, actually, a test setup. I wanted to verify that the attributes had changed, and to do that I wanted to show that they were different and then the same after submission. There's probably a clearer way to do this. I could just remove that line...